lib/v2/luogu/daily.js

```diff
@@ -9,19 +15,30 @@ module.exports = async (ctx) => {
     const $ = cheerio.load(response.data);
     const title = $('head title').text();
 
-    const firstPost = $('.am-comment-main .am-comment-bd').first();
-    const dailyLink = firstPost.find('a').first().attr('href');
-    const issueHeading = firstPost.find('h1').text().trim();
+    const injection_script = $('head script')
+        .filter((_, el) => $(el).text().startsWith('window._feInjection'))
+        .first()
+        .text();
+    // the window._feConfigVersion and window._tagVersion seems to be a constant, but in cast it will change, write a regexp to match it.
+    const JSONRaw = injection_script.replaceAll('window._feInjection = JSON.parse(decodeURIComponent("', '').replace(/"\)\);window._feConfigVersion=(\d*);window._tagVersion=(\d*);/g, '');
```

Comment:
You can use regex to match the encoded JSON instead of manipulating the string:
```js
.match(/window\._feInjection = JSON\.parse\(decodeURIComponent\("(.*?)"\)\);/)[1]
```